RSSDP/SsdpDevice.cs

```diff
@@ -134,11 +135,13 @@ public string FullDeviceType
         {
             get
             {
-                return String.Format("urn:{0}:{3}:{1}:{2}",
-                this.DeviceTypeNamespace ?? String.Empty,
-                this.DeviceType ?? String.Empty,
-                this.DeviceVersion,
-                this.DeviceClass ?? "device");
+                return String.Format(
```

Comment:
Shouldn't this be `string`?

Reply:
Old RSSDP code, out of scope for this PR

Reply:
Old or not, it's still a changed line. Why not just fix it?

Reply:
Because that codebase is full of this stuff, if you want you can change all of them at once.
I haven't because it's always been my plan to change back to upstream as soon as we can.
This change was only to make it compile on .net6